### PR TITLE
Updated Image

### PR DESCRIPTION
**Pull request by joinimran**
Updated the image and removed the date dependency to keep it live longer.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/1742

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md [hunk 1]
```diff
@@ -80,3 +80,4 @@ In environments that have more restrictive internet access, or for those that re
     -   <https://support.office.com/en-us/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2> (includes all Office services, DNS names, IP addresses; includes Azure AD and other services that may overlap with those listed above)
 
 -   **Certificate revocation lists (CRLs).**  Some of these services will also need to check certificate revocation lists (CRLs) for certificates used in the services.  A full list of these is documented in the Office documentation at <https://support.office.com/en-us/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2#bkmk_crl> and <https://aka.ms/o365chains>.
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md]:
```suggestion
-   **Certificate revocation lists (CRLs).**  Some of these services will also need to check certificate revocation lists (CRLs) for certificates used in the services.  A full list of these is documented in the Office documentation at <https://support.office.com/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2#bkmk_crl> and <https://aka.ms/o365chains>.
```

File: windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md [hunk 2]
```diff
@@ -80,3 +80,4 @@ In environments that have more restrictive internet access, or for those that re
     -   <https://support.office.com/en-us/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2> (includes all Office services, DNS names, IP addresses; includes Azure AD and other services that may overlap with those listed above)
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md]:
```suggestion
    -   <https://support.office.com/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2> (includes all Office services, DNS names, IP addresses; includes Azure AD and other services that may overlap with those listed above)
```

File: windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md [hunk 3]
```diff
@@ -80,3 +80,4 @@ In environments that have more restrictive internet access, or for those that re
     -   <https://support.office.com/en-us/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2> (includes all Office services, DNS names, IP addresses; includes Azure AD and other services that may overlap with those listed above)
 
 -   **Certificate revocation lists (CRLs).**  Some of these services will also need to check certificate revocation lists (CRLs) for certificates used in the services.  A full list of these is documented in the Office documentation at <https://support.office.com/en-us/article/Office-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2#bkmk_crl> and <https://aka.ms/o365chains>.
+-	**Hybrid AAD Join.**  Hybrid AAD can be join, the machine should be on corporate network for hybrid AAD join to work. See details at <https://docs.microsoft.com/windows/deployment/windows-autopilot/user-driven-hybrid>
```
Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md]:
```suggestion
-	**Hybrid AAD Join.**  Hybrid AAD can be joined. The computer should be on corporate network for hybrid AAD join to work. For more detaisl see <https://docs.microsoft.com/windows/deployment/windows-autopilot/user-driven-hybrid>.
```

**Comment by mypil** [comment]
@joinimran - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by joinimran** [comment]
Changes applied.

**Comment by mypil** [comment]
@joinimran - there's a branch conflict on this PR, let me loop in @martyav @illfated in case they have some suggestions on how to resolve this.

@martyav @illfated  - do you have any inputs on how we can resolve this branch conflict?

Thank you.

**Comment by illfated** [comment]
Thank you for asking. Myself, I use Git Extensions (https://github.com/gitextensions/gitextensions/blob/master/README.md) to manage my branches, updates and pull requests. (Maybe martyav uses the same, or a similar Git client software). Maybe it is possible to rebase or merge in Github Desktop (https://desktop.github.com/) -- I don't know yet because I have not tested it myself. An other alternative is that I create a pull request directly to @joinimran's fork here on Github, with a rebased file to keep his changes intact (it is recommended to always create a new branch (with a relevant name) before starting to make changes meant for a Pull Request to the originating master branch (`MicrosoftDocs/windows-itpro-docs`).

**Comment by illfated** [comment]
Meh. I just found out that Github Desktop will not be a practical tool for me to use to rebase joinimran's master branch. I will try again, using Git Extensions, which is the tool I know best for this task.

**Comment by illfated** [comment]
Actually, the file `windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md` **no longer exists**, it has been deleted or moved to a new location (or maybe the content has been exported to a different file).

My recommendation is to find the new location and open a PR for that file instead. I must admit that I find it a bit confusing to find, at least on short notice, the precise commit where this file was deleted.

**Comment by illfated** [comment]
Sorry, I could not find the actual commit that removed the file, going back several days in the commit history. If any of you are familiar enough with the commits and want to search for it, be my guest. I can only stand by my recommendation above, to relocate the image change to the new content, wherever it has moved to.

**Comment by mypil** [comment]
Thanks for your inputs @illfated. Looping in the doc owner @greg-lindsay as he may be able to help fix the branch conflict and merge this PR.

@greg-lindsay - PR has been copy edited and is ready for final review. This is based on issue #1742. 

Thank you.

**Comment by illfated** [comment]
Very well, fair enough. The only thing I know is that the source file https://github.com/MicrosoftDocs/windows-itpro-docs/tree/master/windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md gives a 404 because the target file has in fact been removed completely. The only place it still exists is in @joinimran's personal fork of https://github.com/MicrosoftDocs/windows-itpro-docs/. What happened to the original file is still a mystery to me, because I can't seem to find the related commit which removed the file altogether.

**Comment by joinimran** [comment]
@illfated , Let me know what input is required from my side to fix this issue once for all.

**Comment by illfated** [comment]
Thank you for asking. Wish I knew the solution, but it looks like the target file `windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md` has been deleted by one of the Microsoft Docs admins after you opened your pull request. If you check the contents of the directory https://github.com/MicrosoftDocs/windows-itpro-docs/tree/master/windows/deployment/windows-autopilot/ , you will see that there is no file named `windows-autopilot-requirements-network.md` any more. I don't know when and what the maintainers of this directory changed, besides removing the file, because I was unable to find the commit where it happened.

Worst case scenario: You will have to close this PR and open a new one, if you are able to find where the page material has gone. I can try to find it, but my internet connection from home is unstable/unpredictable, so I can't guarantee how soon I will be able to report back any findings to work on.

Best case scenario: One of the admins could post here and inform us about what happened to the file and what they decided to do with the page material, maybe it has been moved somewhere else.

@mypil : Could you ask any of the page authors or current page admins to help us out here, so we can decide what to do with the work already put into this removed file?

**Comment by illfated** [comment]
Maybe @greg-lindsay could inform us about what happened and if we should simply close this PR or if it can be put to good use in one of the other files in the same directory.
@lizap: Do you know anything about when the file `windows-autopilot-requirements-network.md` got removed from this repository, and by which commit?

**Comment by greg-lindsay** [comment]
FYI, reading through the questions, I will provide answers.

https://github.com/MicrosoftDocs/windows-itpro-docs/tree/master/windows/deployment/windows-autopilot/windows-autopilot-requirements-network.md gives a 404 but you should note that https://docs.microsoft.com/windows/deployment/windows-autopilot/windows-autopilot-requirements-network redirects properly to https://docs.microsoft.com/en-us/windows/deployment/windows-autopilot/windows-autopilot-requirements#networking-requirements

The entire Autopilot doc set was reorganized over a month ago (the end of June). Normally if you pull from the master before starting any edits you'll see the changes and avoid merge conflicts.

**Comment by greg-lindsay** [comment]
Not sure why a virus and threat protection image was updated with the autopilot docs. You'll have to submit this as a separate PR.

**Comment by greg-lindsay** [comment]
Not sure why a virus and threat protection image was updated with the autopilot docs. You'll have to submit this as a separate PR.

**Comment by illfated** [comment]
Thank you for the detailed explanation. I should have remembered, I forgot to check how far behind the PR author's forked repository is, to understand why this is something that did not happen recently.

**Comment by joinimran** [comment]
> Not sure why a virus and threat protection image was updated with the autopilot docs. You'll have to submit this as a separate PR.

@greg-lindsay Let me explain this because I also want to understand this, for networkpath.md file I have created a PR long ago with the desktop version and don't know how the file was moved/deleted. Later on when I have updated the image from the deskotp version and the desktop created interdependencies of both commits in one PR. This was the reason that virus update file gets linked with network-path.md file. I don't know how does that happened and what was the reason. If there is any info or resource to understand this will be really helpful.

**Comment by illfated** [comment]
A similar thing happened to my local repository, where I had to reset file/folder changes, because the image https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/identity-protection/hello-for-business/images/pinerror.png showed up as a non-committed change in my updated local cloned repository on my computer (even if all I did was to refresh my local repository clone by copying the repository contents over from a USB drive after I had lost internet connection at home).

**Pull request closed without merging**